languages/en.php

```diff
@@ -1287,7 +1289,7 @@
 	'entity:delete:success' => 'Entity %s has been deleted',
 	'entity:delete:fail' => 'Entity %s could not be deleted',
 	
-	'entity:can_delete:invaliduser' => 'Can not check canDelete for user_guid [%s] as the user does not exist.',
+	'entity:can_delete:invaliduser' => 'Cannot check canDelete for user_guid [%s] as the user does not exist.',
```

Comment:
Let's change ```canDelete``` to ```canDelete()```. I remember someone assuming that it was some kind of a typo for "can delete".